firefox-ios/firefox-ios-tests/Tests/XCUITests/CreditCardsTests.swift

```diff
@@ -189,9 +190,9 @@ class CreditCardsTests: BaseTestCase {
         } else {
             nameOnCard = app.otherElements.textFields.element(boundBy: 1)
         }
-        nameOnCard.tap()
+        nameOnCard!.tap()
```

Comment:
We should not use this... I know it happens when you have moved the definition outside but I would rather use a different approach here. I am running this locally to see other options similar to how we have the rest of the tests